waddrmgr/db.go

```diff
@@ -859,6 +860,9 @@ func fetchLastAccount(ns walletdb.ReadBucket, scope *KeyScope) (uint32, error) {
 	metaBucket := scopedBucket.NestedReadBucket(metaBucketName)
 
 	val := metaBucket.Get(lastAccountName)
+	if val == nil {
```

Comment:
Why wasn't this needed before? Or is this for the case where we create the new wallet as pure watch-only?

Reply:
I think this only occurs when the wallet is created as pure watch-only.

Reply:
Ah, because no default accounts are created, right.